css/style.css

```diff
@@ -4,16 +4,163 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0px;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
 
+.links {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    
+    }
 
+a {
+    margin: 20px;
+    text-decoration: none;
+    color:grey;
+}
+
+
+a:hover {
+    color: darkorange;
+}
+
+.logo {
+  display: inline-flex;
+  height: auto;
+  width: 50%;
+  margin:5px;
+
+    }
+
+
+.container{
+    background-image: url(../img/first-background.jpg);
+    padding: 250px;
+    align-content: center;
+    background-size: cover;
+    max-width: 100%;
+   }
+
+
+
+.insidecon {
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    color: whitesmoke;
+    margin-bottom: 40px;;
+}
+
+
+
+.btn  {
+    background: #FF5F1F;
+    border-radius: 5px;
+    border:none;
+    height: 40px;
+    width: 120px;
+    margin: 10px;
+}
+
+
+.btn:hover {
+    background: black;
+}
+
+
+.parent {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+      
+}
+
+    
+
+.seccontainer {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    gap: 120px;
+    margin:auto;
```

Comment:
you can actually add `justify-content:center` and remove the `.parent` class (and div) completely and it works as intended!

Reply:
Amended thank you